core/storage/question/gae_models.py

```diff
@@ -153,6 +153,8 @@ class QuestionSkillLinkModel(base_models.BaseModel):
     question_id = ndb.StringProperty(required=True, indexed=True)
     # The ID of the skill to which the question is linked.
     skill_id = ndb.StringProperty(required=True, indexed=True)
+    # The difficulty of the skill.
+    skill_difficulty = ndb.FloatProperty(required=False)
```

Comment:
Actually shouldn't this be required=True?

Might as well add indexed=True too.

Reply:
Done